type_data/tables/aggressive_npc.yaml

```diff
@@ -7,6 +7,67 @@ prefix:
   - veteran
   # confluence
   - huge
+  # boon / glam / boss creatures
```

Comment:
These should exist in a migration file. the `.yaml` files are only here to serve as a baseline of the initial GameObj import into the ruby DSL